app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<h2 class="public-recipe-h2"> Public Recipes </h2>
+<% @public_recipes.each_with_index do |public_recipe, index| %>
+  <%= link_to recipe_path(public_recipe[:id]), class: "each-public-recipe text-decoration-none" do %> 
+    <div class="row border public-recipe-2c">
+      <div class="col-3">
+        <h3> <%= public_recipe[:name] %> </h3>
+       <p class="public-recipe-user"> By: <%= User.find_by(email: public_recipe[:email]).name %> </p>
```

Comment:
- [ ] @jugosack and @lukman155 great work on your recipes and displaying them, however as shown below I was unable to see the the recipe owner in public recipes page 
![image](https://github.com/jugosack/RecipeAppRails/assets/61408860/ddc667ad-6ae0-455c-b007-4e0ad9e09a67)